prow/config/config.go

```diff
@@ -1194,6 +1211,12 @@ func (c *Config) validateComponentConfig() error {
 		if _, err := url.Parse(v); err != nil {
 			return fmt.Errorf(`Invalid value for Planks job_url_prefix_config["%s"]: %v`, k, err)
 		}
+		// TODO(@sbueringer): Remove in September 2020
+		if strings.HasSuffix(v, "gcs/") {
+			logrus.Warning("configuring the 'gcs/' storage provider suffix in the job url prefix is now deprecated, " +
```

Comment:
FYI: I'm personally a fan of stirings.join over + for long strings

strings.Join([]string{
    "line 1",
    "line 2",
}, "\n") // or "" instead of "\n" if we just want one long string
```

Reply:
Thx, makes sense, fixed it and rebased onto current master